resources/views/dashboard/admin/files/upload.blade.php

```diff
@@ -50,4 +55,37 @@
         </div>
     {{ html()->form()->close() }}
 </div>
+
+<script>
```

Comment:
Script looks fine, but let's move it to a .js asset and link it back into the blade. We made a big effort to remove all js from the blades about a year ago. See `resources/views/dashboard/controllers/files.blade.php` for an example on how to do this. Just move the script to it's own .js file located in `resources/assets/js/mix/whatever.js` and then link it at the bottom of the blade like: `<script src="{{mix('js/whatever.js')}}"></script>`